### DISCUSSION TITLE
Stremio plugin?

### DISCUSSION THREAD

**Discussion by n1trux** [jellyfin/jellyfin#6621] [General]
Stremio is a platform to watch Netflix, Amazon Prime etc. and they have an SDK for plugins:

https://github.com/stremio/stremio-addon-sdk

Is it possible to create a Jellyfin plugin for Stremio?

**Comment by rallisf1** [comment]
The link you provided is for building plugins for the stremio app, not the other way around. What is possible is to create jellyfin plugins similar to what stremio has (e.g. Netflix, YTS etc), but as far as I know the project is mainly focused on offline content instead of streaming services which tbh you can already use through their own apps/website.

**Reply by wertercatt** [reply]
I believe OP is wanting the ability to bring their streaming services into their Jellyfin library to use, for example, SyncPlay.

**Comment by akarazniewicz** [comment]
I've build simple, local stremio addon to integrate Jellyfin and Stremio: https://github.com/akarazniewicz/stremio-jellyfin

**Reply by drizzt09** [reply]
Interesting option except developer doesn't respond to his git
